lib/nodejs/vwf.js

```diff
@@ -139,19 +141,37 @@ function HandleFileRequest( request, response ) {
 // directory structure.
 // If all that fails, serve up a 404 response since the request was not handled.
 function Serve( request, response ) {
+    lobby.post( "/sim", function( req, res ) {
```

Comment:
Yes, I think that's a good idea. Be sure to consider where it goes relative to authentication. If it goes after, you would need to send authentication data in with the request (which is correct, but maybe ok to defer).